scripts/qa_crowbarsetup.sh

```diff
@@ -3306,7 +3317,7 @@ function onadmin_proposal
     safely oncontroller check_crm_failcounts
     # For all remaining proposals, check for HA failures after each deployment
     for proposal in horizon ceilometer heat manila trove \
-        designate barbican magnum sahara aodh tempest; do
+        designate octavia barbican magnum sahara aodh tempest; do
```

Comment:
octavia depends on barbican. Is the order important here?

Reply:
good point, fixed.